Plan/api/src/main/java/com/djrapitops/plan/delivery/web/resolver/request/Request.java

```diff
@@ -94,6 +100,36 @@ public URIQuery getQuery() {
         return query;
     }
 
+    /**
+     * Get the raw body, if present.
+     *
+     * @return byte[].
+     */
+    public byte[] getRawBody() {
+        return body;
+    }
+
+    /**
+     * Get the body as an url-encoded form, if present.
+     *
+     * @return {@link URIQuery}.
+     */
+    public URIQuery getFormBody() {
```

Comment:
This should be moved outside of `api` module. A separate class inside the `common` module with a single method that reads the bytes into URIQuery might be appropriate

Reply:
Move the whole method or just the logic in it? As in, to keep the `#getFormBody()` instance method or not

Reply:
Move the whole method